frontend/components/TableContainer/TableContainer.tsx

```diff
@@ -221,13 +221,14 @@ const TableContainer = ({
     prevSearchQuery,
   ]);
 
-  const displayCount = useCallback((): number => {
+  // TODO: refactor?
```

Comment:
To not use `typeof`?

Reply:
@lukeheath, I think we'll need to figure out our preferred approach and refactor to deal with server loading and error state. Right now we have the `displayCount` method and also the `TableContainerUtils.generateCountResultsText` method but neither is a great fit for what I think we want to do here. I've got a few thoughts on how we could do it and it seems like it would be good for the two of us to connect to talk it through so we move this component towards a well-considered and sustainable pattern for handling the count functionality. Do you have some time today or tomorrow?

Reply:
Sounds good! How about Wednesday? Feel free to put something on my calendar.